scripts/import/csip_weps_ifc.py

```
@@ -0,0 +1,89 @@
+
+from pathlib import Path
+
+import pandas as pd
+import requests
+from pyiem.database import get_sqlalchemy_conn, sql_helper
+from tqdm import tqdm
+
+
+def get_cokey(lon: float, lat: float) -> str | None:
+    """Fetch what it has."""
+    resp = requests.post(
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)